bundle/compliance/policy/gcp/iam/ensure_policy_not_managed_by_user.rego

```diff
@@ -0,0 +1,13 @@
+package compliance.policy.gcp.iam.ensure_policy_not_managed_by_user
+
+import data.compliance.policy.gcp.data_adapter
+import future.keywords.every
+import future.keywords.if
+
+default is_policy_not_managed_by_user = false
```

Comment:
nit: double negation hurts readability in general IMO

Reply:
agreed, but that's what the rule is for. adding a utility for `is_managed` and only using it with `not` seemed liked overkill. at least it's explicit